alibi/explainers/partial_dependence.py

```diff
@@ -489,68 +463,45 @@ def _partial_dependence(self,
 
         deciles, values, features_indices = [], [], [],
         for f in features:  # type: ignore[union-attr]
-            # Note that we are using the _safe_indexing procedure implemented in sklearn to retrieve the column.
-            # This is because the _safe_indexing supports many data types such as sparse matrix representation,
-            # pandas dataframes etc. In case we would like to enhance alibi in the future to support other
-            # data types than np.ndarrays, this can be useful.
-            f_indices = np.asarray(_get_column_indices(X, f), dtype=np.int32, order='C').ravel()
-            X_f = _safe_indexing(X, f_indices, axis=1)
-
+            # extract column
+            X_f = X[:, f]
```

Comment:
Perhaps it's worth leaving a `TODO` comment here with the option of using `_safe_indexing` in the future to support more input types?